dll/win32/netid/lang/ro-RO.rc

```diff
@@ -2,93 +2,93 @@
-    PUSHBUTTON "A&nulează", 2, 209, 104, 50, 14
-    LTEXT "Acest nume este utilizat pentru interoperabilitatea cu servicii mai vechi.", 13, 7, 88, 253, 8
+    DEFPUSHBUTTON  "OK", 1, 153, 104, 50, 14
+    PUSHBUTTON "Revocare", 2, 209, 104, 50, 14
+    LTEXT "Acest nume se utilizează pentru interoperabilitate cu computere şi servicii mai vechi.", 13, 7, 88, 253, 8
 END
 
 STRINGTABLE
 BEGIN
     1 "* Necunoscut *"
-    2 "GRUPDELUCRU"
-    3 "La citirea domeniului de apartenență a apărut următoarea eroare:"
-    4 "Schimbarea identității calculatorului"
+    2 "WORKGROUP"
+    3 "La încercarea de a citi informaţiile de membru al domeniului a avut loc următoarea eroare:"
+    4 "Modificări ale numelui computerului"
     5 "Grup de lucru:"
     6 "Domeniu:"
-    7 "Numele de domeniu ""%1"" nu se conformează cu specificațiile Serviciului de nume de domenii de internet, cu toate că este un nume ReactOS legal. Trebuie să folosiți serverul DNS ReactOS pentru numele non-standard."
-    8 "Numele de domeniu ""%1"" nu este formatat corect. Punctele (.) sunt folosite la domeniile separate. Fiecare domeniu este limitat la 63 de caractere. De exemplu: domeniul-1.reactos.org."
-    10 "Noul nume de calculator ""%1"" este prea lung. Numele nu poate fi mai lung de 63 de caractere."
-    11 "Noul nume de calculator introdus nu e formatat corect. Numele standard pot conține litere (a-z, A-Z), numere (0-9), și cratime (-), dar nu spații sau puncte (.). Numele nu poate fi compus numai din cifre."
-    22 "Bun venit în grupul de lucru „%1”."
-    23 "Bun venit în domeniul „%1”."
-    24 "Repornirea calculatorului este necesară pentru a aplica schimbările aduse."
-    25 "Puteți schimba numele și apartenența acestui calculator la un domeniu sau grup de lucru. Schimbările pot afecta accesul la resursele rețelei."
-    1021 "Notă: Doar administratorii pot schimba identitatea calculatorului."
-    1022 "Notă: Identitatea calculatorului nu poate fi schimbată deoarece:"
-    1029 "Noul nume de calculator ""%1"" e un număr. Numele nu poate fi un număr."
-    1030 "Noul nume „%1” conține caractere nepermise. Caracterele nepermise sunt ` ~ ! @ # $ %% ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / și ?"
-    1031 "Primul domeniu al numelui de domeniu ""%1"" este un număr. Primul domeniu nu poate fi un număr."
-    1032 "Numele de domeniu ""%1"" conține caractere ce nu sunt permise. Numele standard de DNS pot conține litere (a-z, A-Z), numere (0-9), și cratime, dar nu spații. Punctele (.) sunt folosite la domeniile separate. De exemplu: domeniu-1.reactos.org."
+    7 "Numele domeniului ""%1"" nu este conform specificaţiilor serviciului Internet Domain Name, deşi este un nume ReactOS legal. Pentru nume nestandard utilizaţi serverul DNS ReactOS."
+    8 "Numele de domeniu ""%1"" nu este formatat corespunzător. Punctele (.) sunt utilizate pentru a separa domeniile.  Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
```

Comment:
```suggestion
    8 "Numele de domeniu ""%1"" nu este formatat corespunzător. Punctele (.) sunt utilizate pentru a separa domeniile. Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
```
This suggestion can just be taken as is.
One question: There was no %2 in here before. But now there is. Does it exist in en-US as well?

Reply:
No, it isn't in ReactOS. I accidentally added it. I'll fix it.

Reply:
Resolved.